includes/class-sensei-modules.php

```diff
@@ -94,6 +94,7 @@ public function __construct( $file ) {
 		// store new modules created on the course edit screen
 		add_action( 'wp_ajax_sensei_add_new_module_term', array( 'Sensei_Core_Modules', 'add_new_module_term' ) );
 		add_action( 'wp_ajax_sensei_get_course_modules', array( $this, 'ajax_get_course_modules' ) );
```

Comment:
Although we need to keep the deprecated function around for awhile in case others are calling it, I don't think we need to continue calling it ourselves, or do we? Otherwise we'll have deprecation notices showing up in the logs.

That also made me wonder if there's any more code to remove related to the first parameter. I went looking for other instances of `sensei_get_course_modules` but didn't find any, which made me wonder if this hook every actually did anything in the first place. 🤔 

One final step in the deprecation process is to add the deprecated code to an issue that is attached to the milestone in which it will be removed. Based on our deprecation policy, that will be in 5.0. There's already an [issue](https://github.com/Automattic/sensei/issues/2976) created for that, so you can add your's there.

Reply:
> Although we need to keep the deprecated function around for awhile in case others are calling it, I don't think we need to continue calling it ourselves, or do we?

I've left the ajax hook just in case someone else is making requests to it. Otherwise, we are not calling it anymore. There was some old JS code calling the action, but I've replaced it with the new action. Let me know if I should remove it.

>That also made me wonder if there's any more code to remove related to the first parameter. I went looking for other instances of sensei_get_course_modules but didn't find any, which made me wonder if this hook every actually did anything in the first place. 🤔

There was some code [that I've removed](https://github.com/Automattic/sensei/pull/4421/files#diff-99b5acd02bfdd8020b116bd2c4a896a110dc852033844c6d4c1a4c120b8f3be2L219) which was calling the hook, but for some reason, that code was not working. Basically, you are correct - the hook didn't do anything. Maybe it was working before the blocks were introduced.

>One final step in the deprecation process is to add the deprecated code to an issue that is attached to the milestone in which it will be removed.

Done, thanks!

Reply:
> I've left the ajax hook just in case someone else is making requests to it.

Good call! I wonder if we should leave a note in #2976 so we don't forget to remove the associated hook when we remove the function?

Reply:
Makes sense. Done!